### ISSUE TITLE
Snapserver crashes when opening json port in a browser

### ISSUE THREAD

**Issue by mueller-ma** [badaix/snapcast#264]

**Comment by badaix** [comment]
how to reproduce this?
Chrome closes the connection with `ERR_INVALID_HTTP_RESPONSE`  
Firefox keeps trying to load the "page", keeping the connection opened  
No crash (current master b1edc34f6e)

**Comment by mueller-ma** [comment]
I couldnt really reproduce it again, but when I access port 1704 via Firefox the playback stops for a seconds and it writes to console
```
Message from syslogd@raspberry at Oct 10 13:32:19 ...
 snapserver[16029]: received message of type 17735 to large: 1651472244#012rece

Broadcast message from systemd-journald@raspberry(Tue 2017-10-10 13:32:19 UTC):

snapserver[16029]: received message of type 17735 to large: 1651472244
rece

```

**Comment by badaix** [comment]
The crash on the control port might be solved with the fix of #267  
I also could reproduce and fix (08d39564e) the crash on the streaming port.

**Comment by mueller-ma** [comment]
Thanks for fixing
